generators/app/templates/graphQL/app/graphql/users/type_defs.js

```diff
@@ -0,0 +1,54 @@
+const { gql } = require('apollo-server');
```

Comment:
remove the user things, we just need templates

Reply:
*only

Reply:
ok